docs/dev/building-on-zksync/contracts/contract-development.md

```diff
@@ -0,0 +1,39 @@
+# Smart contract development
+
+zkSync Era allows developers to build projects using the same programming languages and tools used to build on Ethereum.
+We provide our compiler toolchain customized and optimized for development on zkSync. For more information, see
+[the toolchain documentation](../../compiler-toolchain/overview.md).
```

Comment:
"the compiler toolchain documentation"

Reply:
Would repeat the previous line. Sure?

Reply:
good point.. .missed it. Maybe just "the documentation"

Reply:
I'd prefer keeping `toolchain documentation`, as just `documentation` could refer to any noun of the previous sentence.

Reply:
ok